recipes-bsp/imx-vpu/imx-vpu_5.4.38.bb

```diff
@@ -22,14 +22,13 @@ inherit fsl-eula-unpack
 PLATFORM_mx6 = "IMX6Q"
 
 do_compile () {
-    INCLUDE_DIR="-I${STAGING_KERNEL_DIR}/include/uapi -I${STAGING_KERNEL_DIR}/include"
-
+    INCLUDE_DIR="-I${STAGING_INCDIR}/imx"
     oe_runmake CROSS_COMPILE="${HOST_PREFIX}" PLATFORM="${PLATFORM}" INCLUDE="${INCLUDE_DIR}" all
 }
 
 do_install () {
     oe_runmake PLATFORM="${PLATFORM}" DEST_DIR="${D}" install
 }
 
-PACKAGE_ARCH = "${MACHINE_ARCH}"
+PACKAGE_ARCH = "${MACHINE_SOCARCH}"
```

Comment:
This line is redundant to MACHINE_SOCARCH_FILTER.

Reply:
You're right ... however, it does not hurt. 

In fact, after we finish going over the recipes, I will rework them one more time, globally, to avoid duplication but first I wish to go over the minimal set of changes which allow use to prove it keeps all working.